desktop/core/src/desktop/conf.py

```diff
@@ -1788,6 +1788,13 @@ def get_instrumentation_default():
   help=_('Choose whether to pick configs only from [desktop] [[cluster]]')
 )
 
+ENABLE_UNIFIED_ANALYTICS = Config(
```

Comment:
Also add the config flag in `pseudo-distributed.ini.tmpl` and `hue.ini`?

Reply:
When too CLDR specific, often we skip it

Reply:
Ohh I see!